static/images/mkfavico.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+cd "${0%/*}"
```

Comment:
I would appreciate a small couple-word comment above this line explaining what it does